### ISSUE TITLE
The following query makes the influxdb process crash

### ISSUE THREAD

**Issue by xiam** [influxdata/influxdb#7315]
### Bug report

__System info:__ Linux, InfluxDB 1.0.0

__Steps to reproduce:__

The following query produces a panic and crashes the influxdb process:

```
> select tagKey, foo from _tagKeys limit 1;
ERR: unexpected EOF
```

InfluxDB logs report:

```
panic: runtime error: index out of range

goroutine 828 [running]:
panic(0xba6da0, 0xc82000e040)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/influxdata/influxdb/influxql.auxIteratorFields.send(0xca066a7370, 0x2, 0x2, 0x7f49bcfc9278, 0xc9f6688240, 0x0)
        /root/go/src/github.com/influxdata/influxdb/influxql/iterator.go:474 +0x8fc
github.com/influxdata/influxdb/influxql.(*floatAuxIterator).stream(0xc9ff9b8900)
        /root/go/src/github.com/influxdata/influxdb/influxql/iterator.gen.go:891 +0x1d7
created by github.com/influxdata/influxdb/influxql.(*floatAuxIterator).Start
        /root/go/src/github.com/influxdata/influxdb/influxql/iterator.gen.go:844 +0x35
```

__Expected behavior:__ Return an error

__Actual behavior:__ Server crashed

**Comment by jsternberg** [comment]
Please use `SHOW TAG KEYS` instead of directly using the select query. The select query isn't part of a public interface which is why the measurement name starts with an underscore.

**Comment by xiam** [comment]
Thanks. Yes, I am using `SHOW TAG KEYS`, I was just playing around with `_tagKeys` and found that dangerous query.

**Comment by jwilder** [comment]
Fixed via #7328
